### DISCUSSION TITLE
megane 3, fluence seat belt mute

### DISCUSSION THREAD

**Discussion by bilo300** [cedricp/ddt4all#1250] [Q&A]
hello friends. how to turn off the seat belt sound. i watched a few videos. in the ddt4all program it is turned off in the TdB_J95_V0302_Valise tab. i do not have this tab. can you help me. thanks

**Comment by AndyIreland** [comment]
try download from here - its database included seems to have that ecu listed - choose 'all cars'  to download
https://www.drive2.ru/b/498093336985338243/

**Reply by bilo300** [reply]
thank you very much my friend. I will test it

**Comment by AndyIreland** [comment]
its definitely there - however its only on the 'read' side:
![image](https://github.com/user-attachments/assets/9fa86d11-213e-4211-a1e4-ed2518e2cf37)

so you may have to use 'manual' datawrite command (be careful!)
![image](https://github.com/user-attachments/assets/8cec7379-dac9-4519-8c67-32146430a252)
